packages/assets-controllers/src/multi-chain-accounts-service/multi-chain-accounts.ts

```diff
@@ -0,0 +1,52 @@
+import { handleFetch } from '@metamask/controller-utils';
```

Comment:
Nice move with the separate client.  Also good to attempt OpenAPI generation but we can live without it for now.